node_modules/README

```diff
@@ -1,9 +0,0 @@
-# Cockpit Javascript Dependencies
-
```

Comment:
Please put that back. @allisonkarlitskaya has a PR which avoids this problem entirely for the future :balloon: